scripts/mockapi.js

```diff
@@ -35,35 +36,41 @@ function getHostUrl() {
   return hostUrl;
 }
 
-async function fetchRecommendations(type) {
-  const hostUrl = getHostUrl();
-  const apiUrl = `${hostUrl}${apiEndPoints[type]}`;
+async function fetchDataFromAPI(url) {
```

Comment:
This is inline with PR changes here https://github.com/aemsites/icicidirect/pull/96/files

Reply:
yes